lca_activity_browser/app/controller.py

```diff
@@ -1,28 +1,22 @@
 # -*- coding: utf-8 -*-
+from .signals import signals
```

Comment:
This is extreme nitpicking, but it would be nice to follow [pep8](https://www.python.org/dev/peps/pep-0008/#id23) in all of the files. This is all over the place at the moment. Guidelines are to follow this order:
- standard lib
- 3d party module
- local imports
separated by blank line
But no need to fix it here, i'll go over the whole codebase and fix this all in one go when i have time